CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - navigator: Improve navigation by waiting screen changes instead of hardcoded sleeps
 - navigator: navigate_and_compare(): Add optional snap_start_idx parameter
 - navigator: navigate_until_text(): take a list of validation instructions and check last screen
+- conftest: `--log_apdu_file <filename>` option
```

Comment:
1.1.0 is already deployed no? This should be in the `## [Unreleased]` entry.

Reply:
Done